scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -1017,7 +1017,7 @@ class FormatWriter(formatOps: FormatOps) {
     ft.meta.rightOwner match {
       case x: Importer => x.importees.length > 1
       case _ =>
-        val args = getApplyArgs(ft, true)._2
+        val TreeArgs(_, args) = getApplyArgs(ft, true)
```

Comment:
why not
```
getApplyArgs(ft, true).args
```
otherwise, there was no point in creating a case class with _named_ fields.